shared/oae/css/oae.components.css

```diff
@@ -194,6 +194,85 @@ i.fa.large:before {
 }
 
 
+/****************
+ ** SWITCH TAB **
+ ****************/
+
+.oae-switchtab-container {
+    text-align: center;
```

Comment:
Could this remain an implementation detail for where the tab switch is being used? If I wanted them to be centered, I would just apply `text-center` to the container I put the tab switch in. If I want them to be left aligned, I wouldn't.